clients/graphql/client.go

```diff
@@ -100,6 +103,21 @@ func AlertToProto(alert *getAlertsAlertsAlertsResponseAlertsAlert) *protocol.Ale
 		}
 	}
 
+	for i, label := range alert.Labels {
+		if label == nil {
+			logrus.WithField("alert-id", alert.AlertId).Warn("malformed alert label")
+			continue
+		}
+
+		labels[i] = &protocol.AlertEvent_Alert_Label{
```

Comment:
One approach is to add a doc<->proto conversion method in the doc package.   We just have to keep this place in mind when we add/remove properties to labels.

Reply:
which `doc` package are you referring to?